styles/collapsibles/mod.css

```diff
@@ -24,6 +24,7 @@ details[open] > summary {
   margin: -1rem;
   margin-bottom: 1rem;
   background: var(--bg-muted);
+  border-radius: calc(var(--bd-radius) - 1px) calc(var(--bd-radius) - 1px) 0 0;
```

Comment:
Wouldn't it be better to put it directly on `summary` so if the background color of summary is overriden there's no issue too ?

Reply:
Yes! You right! I moved that line in `summary` declaration. 👍